tests/test_mwss_transform.py

```diff
@@ -396,6 +396,11 @@ def test_aggregate_fiveweekly_increase_employees(self):
 
 class TransformTests(unittest.TestCase):
 
+    def test_no_defaults(self):
+        rv = MWSSTransformer.transform({})
+        self.assertIsInstance(rv, OrderedDict)
+        self.assertFalse(rv)
+
```

Comment:
I think it would be good to have some more tests. At least one which had some data in the transform